tutorials/core-concepts/pieces/README.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Reusable Content With Pieces"
+layout: reference
+children:
+  - reusable-content-with-pieces
+  - displaying-pieces-with-widgets
+  - browsing-directory-of-pieces
+  - joins
+---
+
+When you build a website, you could look at it like a puzzle. Each element of the site needs to fit together so you can see the whole picture. In Apostrophe, any kind of structured content is a “piece” and that content becomes pieces of the puzzle that you’re putting together as you build your site. You can use pieces for things ranging from defining a standard format for a blog post to creating a new entity type like “bikes” for a bike shop. Apostrophe uses pieces internally to define different content out of the box – and many other things like users, that you might not think of as “content” – and you can harness that same power to put the pieces of your own puzzle together.
```

Comment:
* Tense issue in the first sentence... how about just "when you build a website, look at it like a puzzle"?
* Second sentence: pages are structured content too, and they are not pieces. In Apostrophe, every document is either a page or a piece. Pages have a fixed location in the page tree, while pieces do not. Pieces can be displayed in one or more places on the site depending on their content, which makes them well suited to content types like products, blog posts, events, etc.
* As for ways Apostrophe uses pieces internally, this is a good idea to talk about, but it'll be clearer if we're specific: Apostrophe uses pieces internally to power the media library ("Images" and "Files" on the admin bar), allowing those things to be reused in many places on the site, as well as things you might not think of as content, like users.